docs/feature-flagging.md

```diff
@@ -0,0 +1,50 @@
+# Feature Flagging
+
+## Using Environment Variables
+
+This guide explains how to update cloud foundry envirionment variables to enable or disable feature flagging.
+
+### Prerequisites
+
+- Cloud Foundry CLI (`cf`) must be installed.
+- Access to Cloud Foundry with the necessary credentials.
+
+## Steps to Update
+
+### 1. Log in to Cloud Foundry
+
+```bash
+cf login -a api.fr.cloud.gov --sso
+```
+
+### 2. Target Correct Space
+
+This should be handled after authenticating but if not, you can target the spaces specifically
+
+```
+cf target -o gsa-tts-benefits-studio -s notify-sandbox
+```
+
+### 3. Set Environment Variable
+
+```
+cf set-env <APP_NAME> <ENV_VAR_NAME> <VALUE>
+```
+
+#### Example:
+
+```
+cf set-env notify-admin-sandbox FEATURE_BEST_PRACTICES_ENABLED true
+```
+
+### 4. Restage the Application
+
+```
+cf restage <APP_NAME>
+```
+
+#### Example:
+
+```
+cf restage notify-admin-sandbox
+```
```

Comment:
This is a great start, thank you! 🎉 

I'm aslo noting here that we'll have to add a bit more to this because the values won't persist between deploys/pushes.

`cf set-env` will only temporarily change it, which is fine for sandbox-based development!  But for any other environment wired up with our CI/CD pipelines (staging, demo, and prod), we'll need to note that if you want the change to persist you have to make the modifications to the environment-specific manifest file and redeploy.

Does this make sense?  It's also a change we can make in a follow-up PR; I just don't want to lose sight of this fact!

Reply:
Oh great point, these are instructions on how to do it in real time quickly, but an additional step needs to be taken to update it in source control.

Reply:
I update this to include updating the environment specific manifest file for source control as the last step!